scalyr_agent/builtin_monitors/tests/linux_process_metrics_test.py

```diff
@@ -17,17 +17,20 @@
 
 import os
 import sys
+import platform
+
+import mock
 
 from scalyr_agent.builtin_monitors.linux_process_metrics import ProcessMonitor
 from scalyr_agent.scalyr_monitor import load_monitor_class
 from scalyr_agent.test_base import ScalyrTestCase
-
-import mock
+from scalyr_agent.test_base import skipIf
 
 __all__ = ["LinuxProcessMetricsMonitorTest"]
 
 
 class LinuxProcessMetricsMonitorTest(ScalyrTestCase):
+    @skipIf(platform.system() == "Darwin", "Skipping Linux Monitor tests on OSX")
```

Comment:
NOTE: The custom decorator we use which supports old Python versions doesn't support decorator the whole class so we need to do it on the method level.